packages/medusa/src/services/claim.ts

```diff
@@ -1,8 +1,51 @@
-import { MedusaError } from "medusa-core-utils"
+import ClaimItemService from "./claim-item"
```

Comment:
Please double-check if your linter is configured correctly; looks like some of the formatting is a bit off :)

Reply:
I just re lint and add the lint on save from the closest location ;)

Reply:
could you have a look at the comment above please :)